### PR TITLE
Port to .NET Standard 1.3

### PR DESCRIPTION
**Pull request by philippe-leonhardt-cko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,231 @@
...
+
+Supporting Docs:
+- [Errors](https://docs.checkout.com/reference/merchant-api-reference/errors/api-validation-errors)
+- [Response Codes](https://docs.checkout.com/reference/response-codes)
+- [Useful Codes](https://docs.checkout.com/reference/useful-codes/country-iso-and-codes)
+
+---
+## API Methods
+
+Each Endpoint has its own service and each service respectively cointains methods.
```
Comment (mikhail-kuzmiankou-cko) [inline README.md]: Typo

File: README.md [hunk 2]
```diff
@@ -379,11 +544,17 @@ catch (Exception e)
...
+## Debug Mode
+
+If you enable the debug mode the HttpRequests and HttpResponses will be logged to console. Set this option to `false` when going Live. Default is `false`.
 
-### Debug Mode
+---
+## Build
 
-If you enable debug mode by setting the **Checkout.DebugMode** to true in the config file or in code all the http requests and responses will be logged in the console.   
+To build the library for the sources, it requires .NET Framework 4.6.1 or later.
```
Comment (mikhail-kuzmiankou-cko) [inline README.md]: Typo

File: README.md [hunk 3]
```diff
@@ -1,70 +1,247 @@
...
-APIClient(string secretKey, Environment env)
-APIClient(string secretKey, bool debugMode)
-APIClient(string secretKey)
+### 2. Configuration
+Before initializing a new instance of the `ApiClient(AppSettings settings)` class, you will be required to **specify the settings**.
+The `AppSettings` object consists of the following settable parameters:
+
+Parameter|Description|Required|Type|Default Value
+:---|---|:---:|---|---
+SecretKey|Your SecretKey is provided to you in The Hub - see [where to find it](https://docs.checkout.com/the-hub/system-administration/business-level-settings#view-api-keys).|true|`string`|`null`
```
Comment (nicolas-maalouf-cko) [inline README.md]: Refer to new docs:
https://docs.checkout.com/docs/business-level-administration#section-view-api-keys

File: README.md [hunk 4]
```diff
@@ -1,70 +1,247 @@
...
 ```
-// Create payload
+... and you are set up to be using our various API endpoints.
+
+>Congratulations for installing and configuring the Checkout .NET Standard Library! If you want to learn what's next, continue reading about our API endpoints.
+
+---
+## Endpoints 
+There are various API endpoints that the `APIClient` interacts with.
+>Make sure you visit our [Docs](https://beta.docs.checkout.com/docs/api-quickstart) if you want to learn in more detail about the endpoints and how they interact. There is even full [examples for transaction lifecycles](https://beta.docs.checkout.com/docs/integration-options) and more details about our fully proprietory solutions that are worth checking out.
```
Comment (nicolas-maalouf-cko) [inline README.md]: New docs URL:
https://docs.checkout.com/docs/api-quickstart

File: README.md [hunk 5]
```diff
@@ -1,70 +1,247 @@
...
+
+Supporting Docs:
+- [Errors](https://docs.checkout.com/reference/merchant-api-reference/errors/api-validation-errors)
```
Comment (nicolas-maalouf-cko) [inline README.md]: https://docs.checkout.com/docs/response-codes#section-api-validation-70-

File: README.md [hunk 6]
```diff
@@ -1,70 +1,247 @@
...
+- [Response Codes](https://docs.checkout.com/reference/response-codes)
```
Comment (nicolas-maalouf-cko) [inline README.md]: https://docs.checkout.com/docs/response-codes

File: README.md [hunk 7]
```diff
@@ -1,70 +1,247 @@
...
+- [Useful Codes](https://docs.checkout.com/reference/useful-codes/country-iso-and-codes)
```
Comment (nicolas-maalouf-cko) [inline README.md]: https://docs.checkout.com/docs/country-codes

File: Checkout.ApiClient.NetStandard/Checkout.ApiClient.NetStandard.csproj [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+    <AssemblyName>Checkout</AssemblyName>
+    <RootNamespace>Checkout</RootNamespace>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <PackageId>Checkout.APIClient</PackageId>
+    <PackageVersion>2.0.0</PackageVersion>
+    <Authors>Checkout.com Integration Team</Authors>
+    <Title>Checkout.APIClient</Title>
+    <Description>
+      API wrapper for the Checkout.com Payment Gateway
+      Docs: https://beta.docs.checkout.com/docs/api-quickstart
```
Comment (nicolas-maalouf-cko) [inline Checkout.ApiClient.NetStandard/Checkout.ApiClient.NetStandard.csproj]: https://docs.checkout.com/docs/api-quickstart

File: README.md [hunk 9]
```diff
@@ -1,70 +1,247 @@
...
+`GetCardToken(TokenCard requestModel)`|`CardTokenCreate`
+
+<br />
+<br />
+
+---
+
+## Examples
+
+Have a look at a few more examples. More detailed information on various Models to be passed in the API Methods are available in our [Merchant API Reference](https://docs.checkout.com/reference/merchant-api-reference/introduction). It is assumed that you already have instantiated your `APIClient`.
```
Comment (nicolas-maalouf-cko) [inline README.md]: https://docs.checkout.com/docs/api-quickstart

**Pull request merged**